src/app/shared/search/import { CaseField } from '..:domain:definition:ca

```diff
@@ -0,0 +1,8 @@
+import { CaseField } from '../domain/definition/case-field.model';
```

Comment:
don't understand what this file is. seems a copy of the one below: src/app/shared/search/search-result-view-item.model.ts